### PR TITLE
Tests cleanup

### PR DESCRIPTION
**Pull request by jdforrester**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -362,12 +365,12 @@ const doTest = (formatversion) => {
...
         data.forEach((test) => {
             idx++;
-            it('Should covert byte range. Test ' + idx,  () => {
+            it(`Should covert byte range. Test ${idx}`,  () => {
                 assert.deepEqual(utils.convertByteClassToUnicodeClass(test[0]), test[1]);
             });
         });
 
-       /* it('Should fetch domains', () => {
+        /* it('Should fetch domains', () => {
```
Comment (Pchelolo) [inline test/index.js]: Oh, that's my fault... I've commented it out for testing and completely forgot.. Could you uncomment it as a part of this PR? It seems like a good piece for this commit message

File: package.json [hunk 2]
```diff
@@ -33,7 +33,6 @@
     "istanbul": "^0.4.5",
     "mocha": "^5.2.0",
     "mocha-eslint": "^4.1.0",
-    "mocha-jshint": "^2.3.1",
```
Comment (Pchelolo) [inline package.json]: You need to remove the reference to this from tests as well

File: test/index.js (hunk 1 above)
Comment (jdforrester) [reply test/index.js]: Done.

File: package.json (hunk 2 above)
Comment (jdforrester) [reply package.json]: Done. :-)

**Pull request merged**